src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -187,18 +191,39 @@ private void OnSurfacePainted(object? sender, SKPaintSurfaceEventArgs e)
 		paint.IsAntialias = true;
 
 		float cornerRadius = (float)_cornerRadius.BottomRight * pixelRatio;
-
+		SKPoint[] cornerRadiusPoints = Array.Empty<SKPoint>();
```

Comment:
I added the initializer to avoid the Compiler Error [CS0165](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0165).

![image](https://github.com/unoplatform/uno.toolkit.ui/assets/116665025/cc1ecd49-db67-43a6-b730-b8463a0ddf53)

Reply:
you can just make the SKShadow.From to taking in null

Reply:
@kazo0 isn't a best practice to use `null` for collections, the `Array.Empty<T>()` is the best here, because it doesn't allocate anything, and the collection will not be `null`.

Reply:
@kazo0 for that case, can we close this review?
Thanks @pictos